### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by Faq** [horsicq/Nauz-File-Detector#10] [Q&A] [answered]
There are new commits time to time, but no [releases](https://github.com/horsicq/Nauz-File-Detector/releases) for a long time. Why so?

**Comment by horsicq** [comment, marked as answer]
Thank a lot for the feedback. I will make a new version in a week.

**Comment by horsicq** [comment]
https://github.com/horsicq/Nauz-File-Detector/releases/tag/0.05

**Reply by graysuit** [reply]
Its hard & time consuming to build and upload binaries. 
Use github ci instead:
https://github.com/horsicq/Nauz-File-Detector/actions
It would compile on every single code change in some seconds.

**Reply by horsicq** [reply]
Thanks. I have to take a look.
